kubernetes/linux/main.sh

```diff
@@ -368,12 +375,12 @@ else
       source defaultpromenvvariables-rs
 fi
 
-#Sourcing telemetry environment variable file if it exists
-if [ -e "telemetry_prom_config_env_var" ]; then
-      cat telemetry_prom_config_env_var | while read line; do
-            echo $line >> ~/.bashrc
+#Sourcing environment variable file if it exists. This file has telemetry and whether kubernetes pods are monitored
+if [ -e "prom_config_env_var" ]; then
+      cat prom_config_env_var | while read line; do
+            echo $line >> /opt/env_vars
       done
-      source telemetry_prom_config_env_var
```

Comment:
Why is telemetry prefix removed from these files? They help differetiate telemetry from non tel

Reply:
I'm using an environment variable from that file (CUSTOM_PROM_MONITOR_PODS) to help determine if the sidecar should be turned off or not. I changed the file to not be named telementry since it isn't just telemetry any more. 

I could write that data to a new file instead if that's more clear? (I'm open to suggestions)

Reply:
Discussed offline, naming will stay the same